api/build.go

```diff
@@ -514,11 +520,35 @@ func GetOrgBuilds(c *gin.Context) {
 	// nolint: gomnd // ignore magic number
 	perPage = util.MaxInt(1, util.MinInt(100, perPage))
 
+	privateRepos, err := database.FromContext(c).GetOrgPrivateRepoList(o)
+	if err != nil {
+		logrus.Errorf("unable to get private repos for org %s : %s", o, err)
+	}
+
+	var excludeList []int64
+
+	for _, rr := range privateRepos {
+		perm, err := source.FromContext(c).RepoAccess(u, rr.GetOrg(), rr.GetName())
+		if err != nil {
+			logrus.Errorf("unable to get user %s access level for repo %s", u.GetName(), rr.GetFullName())
```

Comment:
Should this return the error instead of moving on?

Reply:
If the github api is down or rate limited this will fail safe and hide the private repos. I don't think it would be ideal to fail the whole process because of a github issue.